data/json/items/gunmod/mechanism.json

```diff
@@ -85,5 +85,53 @@
     "install_time": "10 m",
     "min_skills": [ [ "weapon", 2 ] ],
     "flags": [ "WATERPROOF_GUN" ]
+  },
+  {
+    "id": "m249semi",
+    "type": "GUNMOD",
+    "name": { "str": "M249 semi-auto trigger" },
+    "description": "Trigger modification, that allow an m249 to work in semi-auto mode.",
```

Comment:
How about: "A trigger modification that allows the M249 to fire in semi-automatic mode."?